### PR TITLE
update(Enlaces utiles): update links

### LINKED ISSUE #181: Actualizar Enlaces Útiles , sitio en español
Actualizar los enlaces útiles , que sean iguales a los enlaces en ingles

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #181 

### Steps to test
1. Go to Enlaces Útiles inside of Comunidad tag
1. Check content is right

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — DISMISSED** [review]
LGTM!

File: docs/comunidad/enlaces-utiles.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+id: enlaces-utiles
+title: Enlaces Útiles
+sidebar_label: Enlaces Útiles
+---
+
+Aquí hay una lista de enlaces útiles relacionados con la comunidad EOS.
+ 
+
+* [Cómo aprender a desarrollar en EOSIO](https://theblockstalk.medium.com/learning-eosio-development-telos-eos-and-other-blockchains-94d384a8b09f)
```
Comment (leisterfrancisco) [inline docs/comunidad/enlaces-utiles.md:10]: I think could be better put it the question mark in both sides

**Review by steph222 — APPROVED** [review]
💯

File: docs/comunidad/enlaces-utiles.md (hunk 1 above)
Comment (edgarEOS) [reply docs/comunidad/enlaces-utiles.md]: question marks added , thank you

**Review by leisterfrancisco — DISMISSED** [review]
Good work 👍

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by steph222 — DISMISSED** [review]

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Review by steph222 — APPROVED** [review]
😄 Good Job

**Comment by Edgar097** [comment]
Add the other 2 links from kubernetes and telos

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Review by steph222 — APPROVED** [review]

**Pull request merged**